internal/painter/image.go

```diff
@@ -30,17 +30,20 @@ func PaintImage(img *canvas.Image, c fyne.Canvas, width, height int) image.Image
 	case img.File != "" || img.Resource != nil:
 		var file io.Reader
 		var name string
+		isSVG := false
```

Comment:
if anymore vars are added here we could move to;
```suggestion
		var (
			file io.Reader
			name string
			isSVG bool
		)
```